src/main/webapp/timeline.js

```diff
@@ -2,4 +2,55 @@ function addNode() {
     var timeline = document.getElementById("timeline");
     console.log(timeline.innerHTML)
     timeline.innerHTML = timeline.innerHTML.concat("<span class=\"timeline-node\"><>--</span>");
-}
\ No newline at end of file
+}
+
+function loadResults() {
+  fetch('/timeline_data').then(response => response.json()).then((tricks) => {
+    const trickNode = document.getElementById('trick-results');
+    tricks.forEach((trick) => {
+      trickNode.appendChild(createTrickElement(trick));
+    })
+  });
+}
+
+function createTrickElement(trick) {
+  const trickElement = document.createElement('div')
+  trickElement.TrickNode = 'trick-container';
+
+  const trickNameElement = document.createElement('span');
+  trickNameElement.innerText = trick.trick_name;
+
+  const dateElement = document.createElement('span');
+  dateElement.innerText = trick.date;
+
+  const linkElement = document.createElement('span');
+  linkElement.innerText = trick.link;
+
+  const notesElement = document.createElement('span');
+  notesElement.innerText = trick.notes;
+
+  var trickTitle = document.createTextNode("Trick: ");
+  var dateTitle = document.createTextNode("Date: ");
+  var linkTitle = document.createTextNode("Link: ");
+  var notesTitle = document.createTextNode("Notes: ");
+  var br1 = document.createElement("br");
+  var br2 = document.createElement("br");
+  var br3 = document.createElement("br");
+  var br4 = document.createElement("br");
+  var br5 = document.createElement("br");
```

Comment:
I think you could declare a single break element and use it multiple times below

Reply:
I originally did that but nothing printed when I tried appending the same element multiple times in different places. When I did this, though, it worked. I can try again to double check